notify_socket.go

```diff
@@ -193,7 +192,7 @@ func sdNotifyBarrier(client *net.UnixConn) error {
 		return err
 	}
 
-	// Send the write end of the pipe along with a BARRIER=1 message.
+	// send the write end of the pipe along with a BARRIER=1 message
```

Comment:
It's better to have complete sentences (starting with an uppercase letter, ending with a period) as comments. IOW I like the earlier form better.

Reply:
I agree. Changed where there were no previous comments.